### DISCUSSION TITLE
Sidebar issue on Firefox on Android 10 : Arrows in tree are selected (like copy)

### DISCUSSION THREAD

**Discussion by Vultik** [pradyunsg/furo#360] [Potential Issue] [answered]
Here a screenschot, to better understand the issue :
![Screenshot_2022-01-19-15-29-13-891_org mozilla firefox](https://user-images.githubusercontent.com/55872753/150151677-d715d296-4b87-4737-8c60-0630b5cdd12f.jpg)
On Firefox 94.1.2 on Redmi 10, Android 11

**Comment by pradyunsg** [comment, marked as answer]
This was fixed at some point. 😅
